### PR TITLE
Development branch

### PR DESCRIPTION
**Pull request by jmanning58**
Small changes that cleaned up Lab 2

### CODE REVIEW

File: 221/src/Tester.java [hunk 1]
```diff
@@ -1,34 +1,95 @@
...
+                strTwoLength = secondInput.length();
+            }
+        }
+        if (strTwoLength > strOneLength) {
+            for (int j = strOneLength; strTwoLength > j; j++) {
+                firstInput = "0" + firstInput;
+                strOneLength = firstInput.length();
+            }
+        }
+        System.out.println(firstInput + "  a " + secondInput); // Reads correctly---
```
Comment (Smilesc) [inline 221/src/Tester.java]: Move comment to separate line

Comment (jmanning58) [reply 221/src/Tester.java]: Next line has a comment, would you like me to make a new line?

Comment (Smilesc) [reply 221/src/Tester.java]: Yes please

Comment (jmanning58) [reply 221/src/Tester.java]: Moved comment to line above

**Review by Smilesc — APPROVED** [review]
Looks good to me

**Pull request merged**